src/components/molecules/PollBox/PollBox.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback, useMemo } from "react";
+import { useForm, useFieldArray, Controller } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane, faPlus } from "@fortawesome/free-solid-svg-icons";
+import { InputField } from "components/atoms/InputField";
+import { MAX_QUESTIONS_NUMBER } from "settings";
+import "./PollBox.scss";
+
+export type Question = {
+  name: string;
+};
+
+export type PollValues = {
+  topic: string;
+  questions: Question[];
+};
+
+export interface PollBoxProps {
+  onSubmit: (args: PollValues) => void;
+}
+
+const defaultQuestion: Question = { name: "" };
+const defaultValues = {
+  topic: "",
+  questions: [defaultQuestion, defaultQuestion],
+};
+
+export const PollBox: React.FC<PollBoxProps> = ({ onSubmit }) => {
+  const { control, handleSubmit, reset, watch } = useForm<PollValues>({
+    defaultValues,
+  });
+  const { fields, append } = useFieldArray({ name: "questions", control });
+
+  const onCustomSubmit = handleSubmit((data) => {
+    onSubmit(data);
+    reset();
+  });
+
+  const isDisabled = useMemo(() => {
+    const [question1, question2] = watch("questions");
+    return !(watch("topic") && question1.name && question2.name);
+  }, [watch]);
```

Comment:
There's probably no point in `useMemo`'ing this either, since it seems to just be a basic boolean logic/comparison. It'd likely 'cost' us more to memo it than to just let it run.

Reply:
fixed as const